### PR TITLE
Fixes milliseconds precision

### PR DESCRIPTION
**Pull request by gcaracuel**

### CODE REVIEW

File: lib/fluent/plugin/in_gelf.rb [hunk 1]
```diff
@@ -99,7 +99,7 @@ def receive_data(data, addr)
         end
 
         # Use the recorded event time if available
-        time = record.delete('timestamp').to_f if record.key?('timestamp')
+        time = Fluent::EventTime.new(record.delete('timestamp').to_f ) if record.key?('timestamp')
```
Comment (navihtot) [inline lib/fluent/plugin/in_gelf.rb:102]: This is a fix I've been looking for, is it possible to get this merged to official project - https://github.com/MerlinDMC/fluent-plugin-input-gelf ?

Comment (gcaracuel) [reply lib/fluent/plugin/in_gelf.rb:102]: Opened https://github.com/MerlinDMC/fluent-plugin-input-gelf/pull/15 please test this for me :)

Comment (navihtot) [reply lib/fluent/plugin/in_gelf.rb:102]: Hey, sorry it took me long time to write back. Thanks for creating the MR - yes I've tested it - it works for me. I'm actually using your version of the plugin for my purpose. Hope it gets merged into the original project, unfortunately it seems inactive :(

Comment (navihtot) [reply lib/fluent/plugin/in_gelf.rb:102]: actually I'm getting now error on starting fluentd:
```
/usr/local/lib/ruby/gems/2.6.0/gems/fluent-plugin-input-gelf-0.3.0/lib/fluent/plugin/in_gelf.rb:13:in `<module:Plugin>': uninitialized constant Fluent::Plugin::SocketUtil (NameError)
log-collector_1    | Did you mean?  Fluent::SocketUtil
log-collector_1    | 	from /usr/local/lib/ruby/gems/2.6.0/gems/fluent-plugin-input-gelf-0.3.0/lib/fluent/plugin/in_gelf.rb:12:in `<top (required)>'
```

**Pull request merged**